src/raiden_libs/blockchain.py

```diff
@@ -157,15 +156,15 @@ def parse_token_network_event(event: dict) -> Optional[Event]:
 def get_blockchain_events(
     web3: Web3,
     contract_manager: ContractManager,
+    token_network_addresses: List[TokenNetworkAddress],
```

Comment:
The list gets mutated in the function. Should we make that more obvious? Maybe a note in the docstring.

Reply:
Did you change anything about this?

Reply:
I see you did. It's just not in this function but one function further outside.